fe/fe-core/src/main/java/org/apache/doris/statistics/AnalysisJob.java

```diff
@@ -122,7 +122,9 @@ protected void flushBuffer() {
         // buf could be empty when nothing need to do,r for example user submit an analysis task for table with no data
         // change
         if (!buf.isEmpty())  {
-            String insertStmt = "INSERT INTO " + StatisticConstants.FULL_QUALIFIED_STATS_TBL_NAME + " VALUES ";
+            String insertStmt = "INSERT INTO " + StatisticConstants.FULL_QUALIFIED_STATS_TBL_NAME
+                    + "(`id`, `catalog_id`, `db_id`, `tbl_id`, `idx_id`, `col_id`, `part_id`, `count`, `ndv`, "
+                    + "`null_count`, `min`, `max`, `data_size_in_bytes`, `update_time`) VALUES ";
```

Comment:
generate this column names from `org.apache.doris.catalog.InternalSchema#TABLE_STATS_SCHEMA`